eureka-client-archaius2/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'nebula-test-jar'
+
+dependencies {
+    compile project(':eureka-core')
```

Comment:
eureka (1.x) has a weird structure where eureka-client is the base module. eureka-core is really for server side only.